ts/components/menu/items/CopyAccountId/guard.ts

```diff
@@ -1,5 +1,8 @@
 import { PubKey } from '../../../../session/types';
 
+/**
+ * We want to show the copyId for private and not blinded chats only
+ */
 export function showCopyAccountIdAction({
```

Comment:
could be, but I guess it depends, if the guard is used in a bunch of places, it makes sense to have it in a separate file (to avoid cyclic dependencies) 
![Uploading image.png…]()

Reply:
![Screenshot from 2024-12-19 10-39-06](https://github.com/user-attachments/assets/dccadf2b-77bd-4c70-a34a-582092e5c081)